pymc3/plots/forestplot.py

```diff
@@ -123,10 +123,10 @@ def _plot_tree(ax, y, ntiles, show_quartiles, **plot_kwargs):
     return ax
 
 
-def forestplot(trace_obj, varnames=None, transform=identity_transform, alpha=0.05, quartiles=True,
-               rhat=True, main=None, xtitle=None, xlim=None, ylabels=None,
-               chain_spacing=0.05, vline=0, gs=None, plot_transformed=False,
-               **plot_kwargs):
+def forestplot(trace_obj, varnames=None, transform=identity_transform,
+               alpha=0.05, quartiles=True, rhat=True, main=None, xtitle=None,
+               xlim=None, ylabels=None, chain_spacing=0.05, vline=0, gs=None,
+               plot_transformed=False, plot_kwargs=None):
```

Comment:
Why not ** anymore?

Reply:
because forestplot is not accepting arbitrary kwargs, instead is accepting a limited set of arguments as stated in the docstring.  Notice that even when previously we had `**plot_kwargs`, keywords were not really passed.

Reply:
I see. That should probably be updated in the doc string then.

Reply:
I think the changes in this PR are those necessary to conform to the existing docstring. What do you have in mind to change the docsting?

Reply:
I see.